Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39985&attempt_id=bd02ba07-dc1b-4d8a-b3e7-a51d19faa38b)

Attempted: libcanberra, opencv, qstopmotion

<details><summary>Partial log (click to expand)</summary><p>

```
[72/79] Building CXX object CMakeFiles/qstopmotion.dir/src/domain/animation/projectserializer.cpp.o
[73/79] Building CXX object CMakeFiles/qstopmotion.dir/src/frontends/qtfrontend/preferences/transformwidget.cpp.o
[74/79] Building CXX object CMakeFiles/qstopmotion.dir/src/frontends/qtfrontend/preferences/generalwidget.cpp.o
[75/79] Building CXX object CMakeFiles/qstopmotion.dir/src/domain/animation/scene.cpp.o
[76/79] Building CXX object CMakeFiles/qstopmotion.dir/src/frontends/qtfrontend/qtfrontend.cpp.o
[77/79] Building CXX object CMakeFiles/qstopmotion.dir/src/domain/animation/animationproject.cpp.o
[78/79] Building CXX object CMakeFiles/qstopmotion.dir/qstopmotion_autogen/mocs_compilation.cpp.o
ninja: build stopped: subcommand failed.
builder for '/nix/store/09zga2gisdjnmf0gms262kjg5480ib16-qstopmotion-2.3.2.drv' failed with exit code 1
[31;1merror:[0m build of '/nix/store/09zga2gisdjnmf0gms262kjg5480ib16-qstopmotion-2.3.2.drv' failed
```
</p></details>

--- a/pkgs/applications/networking/instant-messengers/baresip/default.nix
+++ b/pkgs/applications/networking/instant-messengers/baresip/default.nix
@@ -1,6 +1,5 @@
-{stdenv, fetchurl, zlib, openssl, libre, librem, pkgconfig
-, cairo, mpg123, gstreamer, gst-ffmpeg, gst-plugins-base, gst-plugins-bad
-, gst-plugins-good, alsaLib, SDL, libv4l, celt, libsndfile, srtp, ffmpeg
+{stdenv, fetchurl, zlib, openssl, libre, librem, pkgconfig, gst_all_1
+, cairo, mpg123, alsaLib, SDL, libv4l, celt, libsndfile, srtp, ffmpeg
 , gsm, speex, portaudio, spandsp, libuuid, ccache, libvpx
 }:
 stdenv.mkDerivation rec {
@@ -11,11 +10,10 @@ stdenv.mkDerivation rec {
     sha256 = "02bf4fwirf3b7h3cr1jqm0xsjzza4fi9kg88424js2l0xywwzpgf";
   };
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [zlib openssl libre librem
-    cairo mpg123 gstreamer gst-ffmpeg gst-plugins-base gst-plugins-bad gst-plugins-good
+  buildInputs = [zlib openssl libre librem cairo mpg123
     alsaLib SDL libv4l celt libsndfile srtp ffmpeg gsm speex portaudio spandsp libuuid
     ccache libvpx
-    ];
+  ] ++ (with gst_all_1; [ gstreamer gst-libav gst-plugins-base gst-plugins-bad gst-plugins-good ]);
   makeFlags = [
     "LIBRE_MK=${libre}/share/re/re.mk"
     "LIBRE_INC=${libre}/include/re"
@@ -26,7 +24,7 @@ stdenv.mkDerivation rec {
     "CCACHE_DISABLE=1"
 
     "USE_ALSA=1" "USE_AMR=1" "USE_CAIRO=1" "USE_CELT=1"
-    "USE_CONS=1" "USE_EVDEV=1" "USE_FFMPEG=1"  "USE_GSM=1" "USE_GST=1"
+    "USE_CONS=1" "USE_EVDEV=1" "USE_FFMPEG=1"  "USE_GSM=1" "USE_GST1=1"
     "USE_L16=1" "USE_MPG123=1" "USE_OSS=1" "USE_PLC=1" "USE_VPX=1"
     "USE_PORTAUDIO=1" "USE_SDL=1" "USE_SNDFILE=1" "USE_SPEEX=1"
     "USE_SPEEX_AEC=1" "USE_SPEEX_PP=1" "USE_SPEEX_RESAMP=1" "USE_SRTP=1"

